pkg/client/tests/remotecommand_test.go

```diff
@@ -221,7 +221,7 @@ func TestStream(t *testing.T) {
 			url, _ := url.ParseRequestURI(server.URL)
 			config := restclient.ContentConfig{
 				GroupVersion:         &schema.GroupVersion{Group: "x"},
-				NegotiatedSerializer: testapi.Default.NegotiatedSerializer(),
+				NegotiatedSerializer: api.Codecs,
```

Comment:
Why is this change necessary?

I know testapi is a mess, but api.Codecs is a global variable we want to get rid of, every new use of it increases our tech debt.